src/app/swap/get-active-loopd.ts

```diff
@@ -0,0 +1,33 @@
+import { getActiveOnchainLnd } from "@services/lnd/utils"
+import { SwapErrorNoActiveLoopdNode } from "@domain/swap/errors"
+import { BTC_NETWORK, getSwapConfig } from "@config"
+
+export const getActiveLoopd = (): LoopdConfig => {
+  const activeNode = getActiveOnchainLnd()
+  if (activeNode instanceof Error) throw SwapErrorNoActiveLoopdNode
+  switch (activeNode.macaroon) {
+    case process.env.LND1_MACAROON: {
+      return LND1_LOOP_CONFIG
+    }
+    case process.env.LND2_MACAROON: {
+      return LND2_LOOP_CONFIG
+    }
+    default: {
+      return LND1_LOOP_CONFIG
```

Comment:
I think we should default in that case?

Reply:
I meant: I think we should error in the default case

Reply:
oic, will add the error